pkg/linter/linter.go

```diff
@@ -102,8 +100,8 @@ func (l Linter) WithInputModules(input *rules.Input) Linter {
 }
 
 // WithAddedBundle adds a bundle of rules and data to include in evaluation.
-func (l Linter) WithAddedBundle(b bundle.Bundle) Linter {
-	l.ruleBundles = append(l.ruleBundles, &b)
+func (l Linter) WithAddedBundle(b *bundle.Bundle) Linter {
```

Comment:
This is an API change. We can do it without this, but it feels right to do it given how this function is used.

Reply:
This is not used in the Rego playground or in any public code on https://sourcegraph.com/search?q=context:global+WithAddedBundle&patternType=keyword&sm=0